src/NServiceBus.Core/InitializableEndpoint.cs

```diff
@@ -17,6 +17,8 @@ class InitializableEndpoint : IInitializableEndpoint
     {
         public InitializableEndpoint(SettingsHolder settings, IContainer container, List<Action<IConfigureComponents>> registrations, PipelineSettings pipelineSettings, PipelineConfiguration pipelineConfiguration, IReadOnlyCollection<IWantToRunWhenBusStartsAndStops> startables)
         {
+            settings.Set<BusNotifications>(new BusNotifications());
+            settings.Set<NotificationSubscriptions>(new NotificationSubscriptions());
```

Comment:
Since features (should be explicit on the feature setup context in the future) needs this during config time we need to set it here since WireUp happens "after configuration is complete". Does this make sense?